src/nvim/getchar.c

```diff
@@ -4560,3 +4626,47 @@ char_u *getcmdkeycmd(int promptc, void *cookie, int indent, bool do_concat)
 
   return (char_u *)line_ga.ga_data;
 }
+
+bool nlua_execute_keymap()
```

Comment:
`nlua` is not the scope, so it should be `map_execute_lua` or something, also needs `void` to make it a prototype.